gke-dask/notebooks/xgboost-gpu-hpo-job-parallel-k8s.ipynb

```diff
@@ -18,283 +18,20 @@
    "id": "19c0c861-9357-48b1-a317-a9df3deb2319",
```

Comment:
<p>This post is titled "with Kubernetes and XGBoost" so we don't need Kubeflow, just the operator.</p><p> </p><p>Happy to leave the Kubeflow mention in but maybe clearly break it out as optional and that we like it because it gives a nice notebook environment to run this notebook within the k8s cluster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>